docs/sources/features/panels/singlestat.md

```diff
@@ -64,7 +63,7 @@ Sparklines are a great way of seeing the historical data related to the summary
 
 <div class="clearfix"></div>
 
-> ***Pro-tip:*** Reduce the opacity on  fill colors for nice looking panels.
+> **_Pro-tip:_** Reduce the opacity on fill colors for nice looking panels.
```

Comment:
```suggestion
> **Pro-tip:** Reduce the opacity on fill colors for nice looking panels.
```

I don't think the emphasis is needed here.

Reply:
ouch, mybad 🙈 I have prettier formatting enabled so it will reformat the `md` code according to the prettier settings of the project.

What is our thoughts regarding this?